app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def tw_following?(id)
```

Comment:
We had it in the Tweet model before we changed the models and then decided it was more of a helper method.